Sources/SwiftGtk/enums/NumberUpLayout.swift

```diff
@@ -0,0 +1,61 @@
+import CGtk
+
+/// Used to determine the layout of pages on a sheet when printing multiple pages per sheet.
+/// 
+/// https://docs.gtk.org/gtk3/enum.NumberUpLayout.html
+public enum NumberUpLayout {
+    case leftToRightTopToBottom
```

Comment:
Add documentation to these cases if possible

Reply:
These only have images for their descriptions in the documentation.